definitions/src/functions.rs

```diff
@@ -582,7 +580,7 @@ extern "system" {
     pub fn SQLGetDescFieldW(
         descriptor_handle: HDesc,
         record_number: SmallInt,
-        field_identifier: Desc,
+        field_identifier: USmallInt,
```

Comment:
Ah, I had based these params off of the enum representations -- i.e. [here](https://github.com/mongodb/mongo-odbc-driver/blob/9adb634cf55f892d13400f21ca7b93e82cbefb2a/definitions/src/desc.rs#L3). Should I update the `repr` to be an `i16` eg? or is it ok that they diverge since we are representing the enum by its contained values (rather than the range of possible values in odbc)

Reply:
uhm, we are using the same enum in SQLColAttribute and there FieldIdentifier is a SQLUSMALLINT.
Descriptor fields can be driver specific.
We don't have specific description fields at the moment.
If we wanted to be able to support negative driver specific descriptor fields, then it would be needed, but I don't think we will. There is more than enough possible values left that the ODBC API isn't already using.
There won't be any issues with converting i16 values to u16, none of the out-of-bound ones will map to ODBC defined values.
I think you can leave the enum  representation as is.